modules/select2_facets/select2_facets.info.yml

```diff
@@ -2,7 +2,7 @@ name: Select2 Facets
 type: module
 description: Provides a Select2 Facet widget.
 package: Search
-core: '8.x'
+core_version_requirement: ^8.8 || ^9
```

Comment:
Does this need to set the minimum core version to `8.8`?

Could we use `core_version_requirement: ^8 || ^9` and keep `core: '8.x'` key too to support all Drupal 8 versions?

Reply:
We remove the usage of entity_get_form_display in this PR. So after that, the module only works with D8.8. We also want to replace jQuery sortable with SortableJS.